src/mgr/DaemonServer.cc

```diff
@@ -176,7 +176,7 @@ bool DaemonServer::ms_verify_authorizer(Connection *con,
     if (peer_type == CEPH_ENTITY_TYPE_OSD) {
       Mutex::Locker l(lock);
       s->osd_id = atoi(s->entity_name.get_id().c_str());
-      dout(10) << __func__ << " registering osd." << s->osd_id << " session "
+      dout(10) << " registering osd." << s->osd_id << " session "
```

Comment:
NIT, ` #define dout_prefix *_dout << "mgr.server " << __func__ << " "`, do not need the start space.